onebusaway-android/src/test/java/org/onebusaway/android/ui/home/map/FocusBannerViewModelTest.kt

```
@@ -0,0 +1,154 @@
+    @get:Rule
+    val mainDispatcherRule = MainDispatcherRule(UnconfinedTestDispatcher())
+
+    private class FakeRouteFavorites : RouteFavorites {
+        override fun favoriteRouteIds(): Flow<Set<String>> = flowOf(emptySet())
+        override suspend fun setFavorite(
+            routeId: String,
+            shortName: String?,
+            longName: String?,
+            url: String?,
+            favorite: Boolean
+        ) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)